satellite/accounting/live/redis.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package live
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"go.uber.org/zap"
+
+	"storj.io/storj/storage"
+	"storj.io/storj/storage/redis"
+)
+
+type redisLiveAccounting struct {
+	log *zap.Logger
+
+	client *redis.Client
```

Comment:
this package is still depending on a redis, but this can be separated also in a different PR